.github/workflows/ci.yml

```diff
@@ -60,10 +61,13 @@ jobs:
         id: cache-taglib
         uses: actions/cache@v2
         with:
-          path: tmp/${{ env.PLATFORM }}/taglib-${{ env.TAGLIB_VERSION }}
-          key: taglib-${{ matrix.os }}-v${{ env.TAGLIB_VERSION }}
+          path: |
+            tmp/${{ env.PLATFORM }}/taglib-${{ env.TAGLIB_VERSION }}
+            tmp/taglib-${{ env.TAGLIB_VERSION }}.tar.gz
```

Comment:
Without this, `rake swig` kept recompiling taglib in spite of the cache hit.

Reply:
Nice catch!